publish/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM java:8
+
+USER root
+
+RUN apt-get install git
+
+RUN useradd -ms /bin/bash geotrellis
+
+USER geotrellis
+
+RUN set -x \
+    && cd /home/geotrellis \
+    && git clone https://github.com/locationtech/geotrellis /home/geotrellis/geotrellis \
```

Comment:
Are we specifically avoiding trying to publish from the current project directory and forcing publish from something thats already on github ?